src/hb-ot-layout-private.hh

```diff
@@ -241,6 +241,7 @@ enum hb_unicode_props_flags_t {
   UPROPS_MASK_GEN_CAT	= 0x001Fu,
   UPROPS_MASK_IGNORABLE	= 0x0020u,
   UPROPS_MASK_FVS	= 0x0040u, /* MONGOLIAN FREE VARIATION SELECTOR 1..3 */
+  UPROPS_MASK_TAG       = 0x0080u, /* TAG characters */
```

Comment:
AFAICS, this doesn't really need to be a separate bit, we could just rename FVS to FVS_OR_TAG. I guess it's up to Behdad whether he prefers to keep them separate for clarity, or merge to leave the other bit free for some future use.